php-frontend/src/main/java/org/sonar/php/tree/symbols/SymbolUsageVisitor.java

```diff
@@ -39,10 +43,26 @@
   @Override
   public void visitCatchBlock(CatchBlockTree tree) {
     for (NamespaceNameTree exceptionType : tree.exceptionTypes()) {
-      QualifiedName fqn = getFullyQualifiedName(exceptionType, Symbol.Kind.CLASS);
-      ClassSymbol classSymbol = classSymbolIndex.get(fqn);
-      ((NamespaceNameTreeImpl) exceptionType).setSymbol(classSymbol);
+      resolveClassSymbol(exceptionType);
     }
     super.visitCatchBlock(tree);
   }
+
+  @Override
+  public void visitNewExpression(NewExpressionTree tree) {
+    ExpressionTree expression = tree.expression();
+    if (expression.is(Tree.Kind.FUNCTION_CALL)) {
```

Comment:
When the constructor does not require arguments, PHP allows instantiation without parentheses (i.e., "new A;"). In that case the expression will directly be of type `Tree.Kind.NAMESPACE_NAME`.

Reply:
Good point! It shows I'm not a PHP developer... 
I amended the last commit to handle that case.